hack/tools.go

```diff
@@ -0,0 +1,22 @@
+// +build tools
```

Comment:
uh oh, a hack/ dir in community 😬 

I won't block on this, but for posterity's sake, why is this needed?

Reply:
The hack dir already exists for the CI scripts. What this file does is track the tools to vendor (recommended method, per https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module )